docs/siem/detections/api/lists/lists-api-overview.asciidoc

```diff
@@ -0,0 +1,47 @@
+[[lists-api-overview]]
+== Lists API
+
+Lists can be used with detection rule <<exceptions-api-overview, exceptions>>
+to define values that prevent a rule from generating alerts. Lists are made up
+of:
+
+* *List containers*: A container for values of the same {es}
+{ref}/mapping-types.html[data type] (such as, `IP`, `keyword`, and `text`).
+* *List items*: The values used to determine whether the exception prevents an
+alert from being generated.
+
+All list items in the same list container must be of the same data type, and
+each item defines a single value. For example, an IP list container, named
+`internal-ip-addresses-southport`, contains five items, where each item defines
+one internal IP address:
+
+. `192.168.1.1`
+. `192.168.1.3`
+. `192.168.1.18`
+. `192.168.1.12`
+. `192.168.1.7`
+
+To use these IP addresses as values for defining rule exceptions, use the
+<<exceptions-api-overview>> to create an
+<<exceptions-api-create-exception-item, exception item>> that references the
+`internal-ip-addresses-southport` list.
+
+NOTE: Lists cannot be added directly to rules, nor do they define the operators
+used to determine when exceptions are applied (`is in list`, `is not in list`).
+Use an <<exceptions-api-create-exception-item, exception item>> to define the
+operator and associate it with an <<exceptions-api-create-container, exception container>>.
+You can then add the exception container to a rule's `exceptions_list` object.
+
+IMPORTANT: Before you can create lists, you must create `.lists` and `.items`
+indices for the {kib} space (see <<lists-index-api-overview>>).
+
+[float]
+=== Kibana role requirements
+
+To create list containers and items, the user role for the {kib} space must
+have:
+
+* the {ref}/security-privileges.html[`manage_api_key` privilege].
```

Comment:
They actually shouldn't need the `manage_api_key` for lists or exception lists but they still need it normally for rules since rules run on a schedule.

So you are fine to remove it here and in exceptions if you have it there